modules/users/client/components/ProfileViewBasics.component.js

```diff
@@ -13,9 +34,87 @@ export default function ProfileViewBasics({ profile }) {
         {(profile.seen) ? t('Online {{date, fromNow}}', { date: new Date(profile.seen) }) : t('Online long ago')}
       </span>
     </div>
+    {profile.locationLiving &&
```

Comment:
Another way to structure this component might be:


```js
const locationLiving = locationLiving => (
  <div className="profile-sidebar-section">
         <i className="icon-fw icon-building text-muted"></i>
         {t('Lives in')}
         <a ui-sref="search.map({location: locationLiving})">
           {locationLiving}
         </a>
  </div>
);

{ profile.locationLiving && this.renderLocationLiving( profile.locationLiving ) }}
```

It just reads different and might make the main code flow (with lots of conditional rendering) little easier to read, but up to you how you see best. :-)

Reply:
I like this way of structuring it.
Now it would cause repeating some code for translating and I suppose @mrkvon discovered a simpler way of including t() function which I would use for that.
So work in process on this, probably till tomorrow.

Reply:
Added render functions.